src/main/java/edu/umich/its/cpm/BoxAuthUserRepository.java

```diff
@@ -55,6 +55,17 @@
 	String findBoxAuthUserAccessToken(String userId);
 	
 	/**
+	 * Find whether the given user's Box refresh token is valid or not
+	 * True if the refresh token is less than 60 days old
+	 * False otherwise
+	 * 
+	 * @param userId
+	 * @return
+	 */
+	@Query("SELECT refreshToken FROM BoxAuthUser u WHERE u.userId = ?#{[0]} and u.refreshTokenCreatedOn is not null and u.refreshTokenCreatedOn > TRUNC(SYSDATE) - 60)")
```

Comment:
To make this testable make the 60 into a property.  If it is set to 1 then it could be tested one a day at least.

For testing it would be better if the check was in hours not days.